src/perception/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 #==============================================================================
 # Dependencies
 #
+if(NOT DART_utils-urdf_FOUND)
+  message(STATUS "Looking for DART COMPONENT: dart-utils-urdf - NOT found, "
+      "to use perception, please install dart-utils-urdf")
+  return()
+endif()
```

Comment:
I just did reminding `find_package(DART REQUIRED COMPONENTS ...)`, but no reason to capitalize it. Will change to lowercase.